src/WebJobs.Script/Diagnostics/OpenTelemetry/OpenTelemetryConfigurationExtensions.cs

```diff
@@ -158,9 +158,7 @@ private static ILoggingBuilder AddDefaultOpenTelemetryFilters(this ILoggingBuild
                 .AddFilter<OpenTelemetryLoggerProvider>("Azure.*", _ => false)
                 // Host.Results and Host.Aggregator are used to emit metrics, ignoring these categories.
                 .AddFilter<OpenTelemetryLoggerProvider>("Host.Results", _ => false)
-                .AddFilter<OpenTelemetryLoggerProvider>("Host.Aggregator", _ => false)
-                // Ignoring all Microsoft.Azure.WebJobs.* logs like /getScriptTag and /lock.
-                .AddFilter<OpenTelemetryLoggerProvider>("Microsoft.Azure.WebJobs.*", _ => false);
```

Comment:
Would this cause customer to lose some important logs from the runtime? Today, I can see log entries from this category in my AI `traces` table. Will that be gone?

Reply:
By removing the filter, we will actually retain these logs, particularly the initialization logs, which we need. The only logs being removed are the `Executing...` and `Executed...` logs. Request telemetry will still contain all the necessary information such as status, error, exception, and duration, making these logs redundant. Customers will save two logs per invocation, and these will still appear in Kusto.